HW1/css/style.css

```diff
@@ -0,0 +1,66 @@
+.largetitle {
+	font-family: Verdana;
+	color: #666666;
+	font-size: 40px;
+	text-align: center;
+	margin-top: 20px;
+}
+
```

Comment:
for a better convention and easy to read, you can put related styles together. like margin padding border; font-family font size text-align color